setup.py

```diff
@@ -38,7 +38,7 @@
 here = path.abspath(path.dirname(__file__))
 
 
-PANDAS = "pandas~=1.2"  # in 1.1 pandas started using fsspec, in 1.2 they fixed a lot of bugs
+PANDAS = "pandas~=1.3"  # to be able to use XMLDataSet and pandas integration with fsspec
```

Comment:
Why not limit the `pandas~=1.3` requirement to the `pandas.XMLDataSet` extra?

Reply:
What would be the benefit of leaving 1.2? Why not set the more recent version by default since it should normally have more bug fixes and features? Or is there something that 1.2 offers that 1.3 doesnt?

Reply:
Hi @deepyaman 

How would i set 2 different pandas requirements? Should i reset `pandas~=1.2` and then have in pandas require?
```
pandas_require = {
    "pandas.CSVDataSet": [PANDAS],
    "pandas.ExcelDataSet": [PANDAS, "openpyxl>=3.0.6, <4.0"],
    "pandas.FeatherDataSet": [PANDAS],
    "pandas.GBQTableDataSet": [PANDAS, "pandas-gbq>=0.12.0, <1.0"],
    "pandas.HDFDataSet": [PANDAS, "tables~=3.6"],
    "pandas.JSONDataSet": [PANDAS],
    "pandas.ParquetDataSet": [PANDAS, "pyarrow>=0.12.0, <4.0.0"],
    "pandas.SQLTableDataSet": [PANDAS, "SQLAlchemy~=1.2"],
    "pandas.XMLDataSet": [PANDAS, "pandas~=1.3", "lxml~=4.6"],
}
```